.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build Website
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-website:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.0'
+        bundler-cache: true
+    - name: Run .github_actions.sh
+      run: |
+        export NOKOGIRI_USE_SYSTEM_LIBRARIES=true
```

Comment:
You are so fast, you can change this to be outside the run block to something like:

```yaml
env:
    NOKOGIRI_USE_SYSTEM_LIBRARIES: true
```

Reply:
I would prefer `export NOKOGIRI_USE_SYSTEM_LIBRARIES=true` as part of the run block as this variable is only needed within this run block.